src/defaultPhrases.js

```diff
@@ -30,6 +30,8 @@ const chooseAvailableEndDate = ({ date }) => `Choose ${date} as your check-out d
 const chooseAvailableDate = ({ date }) => date;
 const dateIsUnavailable = ({ date }) => `Not available. ${date}`;
 const dateIsSelected = ({ date }) => `Selected. ${date}`;
+const dateIsSelectedAsCheckin = ({ date }) => `Selected for check-in. ${date}`;
```

Comment:
Can we s/checkin/startdate and s/checkout/enddate for consistency with this library? We can use checkin/checkout at airbnb but it'd be good to keep the nomenclature consistent within this library.